gradle-versions/src/main/java/com/markelliot/gradle/versions/api/YamlSerDe.java

```diff
@@ -30,7 +30,7 @@
 import java.nio.file.Path;
 
 public final class YamlSerDe {
-    private static final ObjectMapper mapper =
+    public static final ObjectMapper mapper =
```

Comment:
I probably would've kept this private and added a serialize override that accepts a file. Part of the idea of this class is to encapsulate the ObjectMapper